ignite/pkg/sentry/sentry.go

```diff
@@ -0,0 +1,48 @@
+package sentry
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/getsentry/sentry-go"
+
+	"github.com/ignite/cli/v29/ignite/pkg/errors"
+	"github.com/ignite/cli/v29/ignite/version"
+)
+
+const IgniteDNS = "https://bugs.ignite.com"
```

Comment:
Can we pass the URL as a parameter so this pkg can be reusable in other ignite applications, like network, and the const can be in the top level, like inside the service package

Reply:
I would rather not, then you still need to extract the environment and release logic out of this package, making it basically useless.